### ISSUE TITLE
PCF DEV was working and now not starting

### ISSUE THREAD

**Issue by pgerbino** [vmware-archive/pcfdev#293]
Both on my windows and Linux installations on the same box; PCF DEV was working and now fails. I see the following failure:

Process '\''syslog_drain_binder'\''       Execution failed

It is consistently this process on both os's - is there anything I can do to remediate this? Note I dual boot my SSD for this and, originally, it worked on both but now doesn't.  Any help appreciated.

Thanks,

Piero

cf version 6.34.1+bbdf81482.2018-01-17
PCF Dev version 0.28.0 (CLI: 5cda315, OVA: 0.547.0)

**Comment by sjudeng** [comment]
I'm seeing this issue too. It looks like the certificate expired 2018-02-15.

```
$ head -n 7 /var/vcap/sys/log/syslog_drain_binder/syslog_drain_binder.log 
panic: x509: certificate has expired or is not yet valid

goroutine 1 [running]:
panic(0xa34b80, 0xc4201d6180)
	/var/vcap/data/packages/golang1.7/c5de560ab1578122cd22032249fb7f1f287768d4/src/runtime/panic.go:500 +0x1a1
main.main()
	/var/vcap/data/compile/syslog_drain_binder/loggregator/src/syslog_drain_binder/main.go:55 +0xbd5
```

```
$ openssl x509 -in /var/vcap/jobs/syslog_drain_binder/config/certs/syslogdrainbinder.crt -text -noout
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 16412547589211907925 (0xe3c51513322b6755)
    Signature Algorithm: sha1WithRSAEncryption
        Issuer: CN=*.service.cf.internal, C=US, ST=New-York, L=New York, O=Pivotal, OU=PCF Dev/emailAddress=pcfdev-eng@pivotal.io
        Validity
            Not Before: Feb 15 22:35:39 2017 GMT
            Not After : Feb 15 22:35:39 2018 GMT
        Subject: CN=*.service.cf.internal, C=US, ST=New-York, L=New York, O=Pivotal, OU=PCF Dev/emailAddress=pcfdev-eng@pivotal.io
...
```

**Comment by ssisil** [comment]
Fixed in [v0.29](https://github.com/pivotal-cf/pcfdev/releases/tag/v0.29.0)
